.github/workflows/connector-publish.yml

```diff
@@ -64,7 +67,11 @@ jobs:
           cross build --profile ${{ inputs.rust-profile }}  --target ${{ matrix.rust-target }} -p ${{ inputs.package-name }}
       - name: Publish
         run: |
-          cdk publish --public-yes --target ${{ matrix.rust-target }} -p ${{ inputs.package-name }}
+          cdk publish \
+            --public-yes \
+            --target ${{ matrix.rust-target }} \
+            -p ${{ inputs.package-name }} \
+            --readme ${{ inputs.readme_path }}
```

Comment:
The workflow will always specify a README, this may break current CI 🤔 
I would like to just skip if theres no `readme_path` specified by I didn't found a clean way to do it,
thinking if this needs to grow now into its own action package?

Reply:
That's ok. If README is not there, should expect publish to fail

Reply:
I think that it is ok, we just need to create an issue in each repo to update if needed and we should do it before next release

Reply:
Here is the set of issues to track progress:

- https://github.com/infinyon/kafka-connector/issues/37
- https://github.com/infinyon/sql-connector/issues/35
- https://github.com/infinyon/http-source-connector/issues/58
- https://github.com/infinyon/mqtt-connector/issues/35
- https://github.com/infinyon/http-sink-connector/issues/28